eng/Version.Details.xml

```diff
@@ -50,6 +50,11 @@
       <Uri>https://github.com/dotnet/roslyn</Uri>
       <Sha>90e570d763637c4d0d30524674a64b07bb885994</Sha>
     </Dependency>
+    <!-- DOes this need a coherent parent dependency? -->
+    <Dependency Name="Microsoft.CodeAnalysis.NetAnalyzers" Version="5.0.0-rc1.20428.12">
```

Comment:
Always, if possible--sdk has the dependency on roslyn-analyzers:

https://github.com/dotnet/sdk/blob/394fae2553937858c8f05a6158613dcc7f821fba/eng/Version.Details.xml#L176-L179

```xml
    <Dependency Name="Microsoft.CodeAnalysis.NetAnalyzers" Version="5.0.0-rc1.20428.12">
      <Uri>https://github.com/dotnet/roslyn-analyzers</Uri>
      <Sha>e856dc126950463c13226c7437fc7e9fb41442dc</Sha>
    </Dependency>
```

So here, we can do `CoherentParentDependency="Microsoft.NET.Sdk"` to link it up.